Access API/AccessAPI/SignalR/SignalRHandler.cs

```diff
@@ -77,15 +75,15 @@ public async Task LeaveGroup(string groupName)
         public async void AddToGroup(string groupName)
         {
             await Groups.AddToGroupAsync(Context.ConnectionId, groupName);
-            await Clients.Client(suggestorClientId).SendAsync("JoinGroup", groupName);
+            await Clients.Client(_suggestorClientId).SendAsync("JoinGroup", groupName);
             Debug.WriteLine($"Added {Context.ConnectionId} to group: {groupName}");
         }
 
         //Called upon clients disconnects
         public async void RemoveFromGroup(string groupName)
         {
             await Groups.RemoveFromGroupAsync(Context.ConnectionId, groupName);
-            await Clients.Client(suggestorClientId).SendAsync("LeaveGroup", groupName);
+            await Clients.Client(_suggestorClientId).SendAsync("LeaveGroup", groupName);
             Debug.WriteLine($"Remove {Context.ConnectionId} from group: {groupName}");
         }
 
```

Comment:
Ikke her men linje 97 giver 2 checkwarnings af "Nullability of type of parameter 'exception' doesn't match overridden member (possibly because of nullability attributes)."